modules/fedimint-mint-client/src/lib.rs

```diff
@@ -743,6 +743,14 @@ impl ClientModule for MintClientModule {
     }
 }
 
+#[derive(thiserror::Error, Debug, Clone)]
+pub enum ReissueExternalNotesError {
```

Comment:
What if we added a third error type?

```
    #[error("Untyped error")]
    Other(anyhow::Error)
```

Then we could change the return type of `reissue_external_notes()` to `Result<OperationId, ReissueExternalNotesError>` and remove this line:

```/// Can return error of type [`ReissueExternalNotesError`]```

Reply:
I think .downcast is fine, Other cause error chain issues